### ISSUE TITLE
Error after 6th level

### ISSUE THREAD

**Issue by kklocek** [aszady/threelab#1]
When I passed 6th level on browser tab I can see "6 loading ...", and nothing happens. Is it due to the limits of levels of game?

In console I can see

``` javascript
...
Object { x: 6, y: 11, dx: -1, dy: 0, left: 0.75, width: 0.25 }PlayerControls.js:38:3
Object { x: 2, y: 1, dx: 0, dy: 1, left: 0, width: 0.5 }PlayerControls.js:38:3
Object { x: 21, y: 13, dx: 1, dy: 0, left: 0.5, width: 0.5 }
TypeError: mapDescription is undefined
```

**Comment by aszady** [comment]
Thank you for your report.
At this moment, the number of levels is indeed limited.

I see two resolutions for this issue - either create 6th level which is unsolvable or create proper message about winning the game.
Applying the first approach could discourage player from playing this game in the future, as the game contains unsolvable level - and, what player may possibly think, is totally broken. The latter approach isn't also a good one, though. Once a player finishes the game, he won't come back again as well.
Considering current game's architecture, unsolvable level seems to be the easiest solution. On the other hand, message seems to be the best from ux perspective and fair to the user.
I believe that procedural generation of levels could solve all mentioned problems, but requires significantly more effort.

To sum up, there is an issue, but no clear way of solving it.
